frontend/src/components/features/chat/event-content-helpers/get-event-content.tsx

```diff
@@ -35,6 +35,7 @@ export const getEventContent = (
           path: hasPathProperty(event.args) && event.args.path,
           command:
             hasCommandProperty(event.args) && trimText(event.args.command, 80),
+          name: event.action === "call_tool_mcp" && event.args.name,
```

Comment:
@OpenHands it works for Calling MCP tool call, but not for "OBSERVATION_MESSAGE$MCP"

"MCP Tool Result: {{name}}"

Can you modify this field to "mcp_tool_name" and update translation.json accordingly